src/timeline.ts

```diff
@@ -205,18 +205,18 @@ class Timeline implements IAppView {
       .domain([0, timeRange])
       .range([20, widthTimelineDiv-20]); // 20 = Spacing
 
-
     svgtimeline.selectAll('circle')
       .data(items)
       .enter()
       .append('circle')
       .attr('title', (d:any) => (d.time) ? d.time.format(AppConstants.DATE_FORMAT) : d.key)
       .attr('cy', 60)
-      .attr('cx', (d:any) => {
+      .attr('cx', (d:any, i) => {
         if(d.time) {
           return xScaleTime(moment(d.time).diff(moment(items[0].time),'days'));
         } else {
-          return 60;
+          return i * ((widthTimelineDiv-20)/7);
```

Comment:
Please use a constant for the 20. I guess this is the same value as in line 279?
What does the 7 mean? Might be also worth to store in a constant or add a comment next to the line.